mst-content-hint/idlharness.window.js

```diff
@@ -0,0 +1,19 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+// META: script=/webrtc/RTCPeerConnection-helper.js
+
+'use strict';
+
+idl_test(
+  ['mst-content-hint'],
+  ['mediacapture-streams', 'dom'],
+  async idl_array => {
+    idl_array.add_objects({
+      MediaStreamTrack: ['audioTrack', 'videoTrack'],
+    });
+
+    const stream = await getNoiseStream({ audio: true, video: true });
+    self.audioTrack = stream.getAudioTracks()[0];
```

Comment:
Nit: Pull `stream.getAudioTracks()[0]` into a variable.

Reply:
But it has to be on the global object for `add_objects` to work, I presume using `eval()`. Am I misunderstanding?

Reply:
DW - Irrelevant given that there should have been 2 different values :)